core/payment/examples/README.md

```diff
@@ -8,7 +8,7 @@ cd core/payment
 cp ../../.env-template .env
 cargo run --example payment_api
 ```
-To use GNT instead of dummy driver us `cargo run --example payment_api -- --driver=gnt` instead.
+To use Erc-20 instead of ZkSync driver us `cargo run --example payment_api -- --driver=erc20` instead.
```

Comment:
did you test this? i think it also needs platform
The `invoice_flow` also needs to get a matching platform as argument

Reply:
I improved the description above. Thank you for pointing it out.
`payment_api` with `driver` & `platform` parameter seems to start but it took long to get funded - I will let it run - will update whether it works if the experiment will conclude.